### PR TITLE
Make cli handle multiple whitespaces

### PR DESCRIPTION
**Pull request by capkurmagati**
# Which issue does this PR close?
The cli cannot handle whitespace like
```
❯ \?
'\? ' is not a valid command
❯ \quiet true
'\quiet true ' is not a valid command
```
The PR fixes this issue by removing the consecutive whitespaces.

Closes #.

 # Rationale for this change

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

File: datafusion-cli/src/exec.rs [hunk 1]
```diff
@@ -90,7 +90,8 @@ pub async fn exec_from_repl(ctx: &mut Context, print_options: &PrintOptions) {
         match rl.readline("❯ ") {
             Ok(line) if line.starts_with('\\') => {
                 rl.add_history_entry(line.trim_end());
-                if let Ok(cmd) = &line[1..].parse::<Command>() {
+                let command = line.split_whitespace().collect::<Vec<_>>().join(" ");
```
Comment (houqp) [inline datafusion-cli/src/exec.rs:93]: have you tried the trim method? https://doc.rust-lang.org/stable/std/string/struct.String.html#method.trim

**Comment by houqp** [comment]
eventually, we will need to write a proper lexer and parser for pgcli commands :P

File: datafusion-cli/src/exec.rs (hunk 1 above)
Comment (capkurmagati) [reply datafusion-cli/src/exec.rs:93]: The problem I'd like to fix here is not only leading and trailing whitespaces.
Currently, the behavior is like
```
DataFusion CLI v5.1.0

❯ \quiet  true
'\quiet  true' is not a valid command
❯ \pset  format  csv
'\pset  format  csv' is not a valid command
❯ \pset format  csv
Execution error:  csv is not a valid format type [possible values: csv, tsv, table, json, ndjson]
```
Without normalizing whitespaces before parsing commands, every command need to do trim like here.
https://github.com/apache/arrow-datafusion/blob/42f8ab1e44510cea73f9eb2300d28f2fa92ac55a/datafusion-cli/src/functions.rs#L159

Comment (houqp) [reply datafusion-cli/src/exec.rs:93]: oh i see, thanks for the context.

**Review by houqp — APPROVED** [review]

**Review by alamb — APPROVED** [review]

**Comment by alamb** [comment]
Thanks @capkurmagati  -- I think this is a small enough fix we can iterate on it if @Jimexist  has some suggestions as well

**Pull request merged**